### DISCUSSION TITLE
Seq filter log performace lagging

### DISCUSSION THREAD

**Discussion by issarakeawchaom** [datalust/seq-tickets#1907] [Help]
Hello,
I filter with GUID String but search was unsuccessful.

**Comment by liammclennan** [comment]
Hi @issarakeawchaom ,

Can you please provide an example of the your query and what you mean by unsuccessful?

**Comment by issarakeawchaom** [comment]
@liammclennan 
I mean the search takes too long. How should I solve the problem?
![image](https://github.com/datalust/seq-tickets/assets/140586015/a4684630-8f87-4e7f-91a4-54bd86e4df7e)

**Reply by nblumhardt** [reply]
Hi! Thanks for the follow-up. If you can send a screenshot of Data > Storage (the chart, in particular) in All Time view, as well as some information on your server hardware/resources, we should be able to offer some suggestions.

**Reply by issarakeawchaom** [reply]
![image](https://github.com/datalust/seq-tickets/assets/140586015/a938d767-8837-4774-ac6f-9169e268e264)
Resource
![image](https://github.com/datalust/seq-tickets/assets/140586015/8c3fb647-4d08-432e-88b7-c802e52bd1b6)

**Reply by nblumhardt** [reply]
Thanks for the follow-up. That's a very small container - but it's also a small amount of data so you should be able to get it working reasonably with some tweaking 🤞 

In the huge empty gap in the middle of the chart, is there data (even one event per day), or is it completely empty?

If you restrict your query to the last seven days using the "FROM/TO" calendar boxes, does the query complete faster?
